scripts/utils.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021-2023 The Connect Authors
```

Comment:
Do we need this function to be externalized from the script if we delete `scripts/release.js`?